test/integration/nested_model.cc

```diff
@@ -1371,3 +1371,87 @@ TEST(NestedReference, PlacementFrameElement)
     EXPECT_EQ(placementPose, pose.Inverse());
   }
 }
+
+/////////////////////////////////////////////////
+TEST(NestedModel, IncludeElements)
+{
+  const std::string modelRootPath = sdf::filesystem::append(
+      PROJECT_SOURCE_PATH, "test", "integration", "model");
+
+  sdf::setFindCallback(
```

Comment:
nit Consider using non-global version? ;)

Reply:
Using non-global version in https://github.com/osrf/sdformat/pull/509/commits/2631da14dff94ea7c66e2480be626ffd3e33e72e